types/ramda/index.d.ts

```diff
@@ -31,6 +31,7 @@
 //                 Nikita Balikhin <https://github.com/NEWESTERS>
 //                 Wang Zengdi <https://github.com/adispring>
 //                 Marcus Blättermann <https://github.com/essenmitsosse>
+//                 adajoy <https://github.com/adajoy>
```

Comment:
Are you adding yourself intentionally? This adds you as an owner of the package; it's not for crediting contributions (we get that via git). See: https://github.com/DefinitelyTyped/DefinitelyTyped#definition-owners

Reply:
removed